site/documentation/content/admin-guide/amqp-adapter-config.md

```diff
@@ -36,8 +36,8 @@ The following table provides an overview of the configuration variables and corr
 | `HONO_APP_MAX_INSTANCES`<br>`--hono.app.maxInstances` | no | *#CPU cores* | The number of verticle instances to deploy. If not set, one verticle per processor core is deployed. |
 | `HONO_HEALTH_CHECK_BIND_ADDRESS`<br>`--hono.healthCheck.bindAddress` | no | `127.0.0.1` | The IP address of the network interface that the secure port, which exposes the service's health check resources, should be bound to. The HTTPS server will only be started if `HONO_HEALTH_CHECK_BIND_ADDRESS` is set explicitly. |
 | `HONO_HEALTH_CHECK_PORT`<br>`--hono.healthCheck.port` | no | - | The secure port the HTTPS server, which exposes the service's health check resources, should listen on. If set, the adapter will expose a *readiness* probe at URI `/readiness` and a *liveness* probe at URI `/liveness`. |
-| `HONO_HEALTH_CHECK_INSECURE_PORT_BIND_ADDRESS`<br>`--hono.healthCheck.insecurePortBindAddress` | no | `127.0.0.1` | The IP address of the network interface that the insecure port, which exposes the service's health check resources, should be bound to. The HTTP server will only be started if `HONO_HEALTH_CHECK_INSECURE_PORT_BIND_ADDRESS` is set explicitly. |
-| `HONO_HEALTH_CHECK_INSECURE_PORT`<br>`--hono.healthCheck.insecurePort` | no | - | The insecure port the HTTP server, which exposes the service's health check resources, should listen on. If set, the adapter will expose a *readiness* probe at URI `/readiness` and a *liveness* probe at URI `/liveness`. |
+| `HONO_HEALTH_CHECK_INSECURE_PORT_BIND_ADDRESS`<br>`--hono.healthCheck.insecurePortBindAddress` | no | `127.0.0.1` | The IP address of the network interface that the insecure port, which exposes the service's health check resources, should be bound to. |
+| `HONO_HEALTH_CHECK_INSECURE_PORT`<br>`--hono.healthCheck.insecurePort` | no | `8088` | The insecure port of the HTTP server, which exposes the service's health check resources, should listen on. The adapter will expose a *readiness* probe at URI `/readiness` and a *liveness* probe at URI `/liveness`. |
```

Comment:
please apply your changes to the secure port description as well
and don't forget the Auth Server config guide

Reply:
So the default behaviour would be have default values for secure + insecure ports ?

Reply:
good question. So far, we used the port property to control whether a server should be started at all. Now, with having a default port, I guess we could use the availability of key material as a discriminator for which server to start, e.g. if there is key material (and no ports being set explicitly), we start the secure server (using the default port and the key material). If no key material is available (and no ports are set explicitly), we start the insecure server. The default port should be the same for both, though, FMPOV.
WDYT?

Reply:
That sounds reasonable. The only issue i see with this solution is that if the default port is the same for secure and insecure we may end up with mismatches, where the client may expect a insecure connection while the server only accept a secure one.

Reply:
update on this : i tested to start both secure and insecure endpoint with the same port number : both works fine, so this is not an issue. Nice ! :)

Reply:
Are you saying that the health server can be successfully accessed by a client using either HTTP or HTTPS in this case?

Reply:
yes